src/protocols/vnc/vnc.c

```diff
@@ -512,6 +516,15 @@ void* guac_vnc_client_thread(void* data) {
         if (wait_result < 0)
             guac_client_abort(client, GUAC_PROTOCOL_STATUS_UPSTREAM_ERROR, "Connection closed.");
 
+        if (server_runing_start - server_runing_end > GUAC_VNC_IDLE_TIMEOUT) {
+         rfb_client = guac_vnc_get_client(client);
+         server_runing_end = guac_timestamp_current();
+            if(!rfb_client){
+                guac_client_abort(client, GUAC_PROTOCOL_STATUS_UPSTREAM_ERROR, "Connection closed.");
+                break;
+            }
+        }
```

Comment:
There's some odd indentation going on, here. Please correct to follow existing style - 4-tab spaces.

Reply:
Done.